MAUI/Circular-Charts/DataLabels.md

```diff
@@ -141,9 +141,87 @@ this.Content = chart;
 
 ![DataLabel context in MAUI Chart](DataLabel_images/maui_chart_datalabel_context.png)
 
+## Connector line style
+
+This feature is used to customize the connector line. The following [ConnectorLineStyle]() property are used to customize the connector line.
+
+* [Stroke]() – used to change the color of the line
```

Comment:
@EzhilarasanElangovan31 modify the property as - Gets or sets the border stroke color of the connector line.

Reply:
Modified